source/developer/bot-accounts.rst

```diff
@@ -0,0 +1,186 @@
+.. _bot-accounts:
+
+Bot Accounts
+========================================
+
+Use Bot Accounts to integrate with Mattermost through `plugins <https://developers.mattermost.com/extend/plugins/>`_ or the `Mattermost RESTful API <https://api.mattermost.com>`_. Bot accounts access the RESTful API on behalf of a bot through the use of the :doc:`personal access tokens feature <personal-access-tokens>`.
+
+// TODO screenshot. Waiting for bug fix where channel header is missing from bot DMs
+
+Bot accounts are just like user accounts, except they:
+
+  - Cannot be logged into, and bypass all restrictions related to authentication, such as multi-factor authentication (MFA) and custom terms of service.
```

Comment:
> bypass all restrictions related to authentication, such as multi-factor authentication (MFA) and custom terms of service

No they don't. Those things just don't apply to anything but regular logins which bots can't do anyway.

Reply:
Updated to just `Cannot be logged into.`